### PR TITLE
Added Pagination Props as example

### PR DESCRIPTION
**Pull request by blalan05**
## Description
Added Pagination Props as an example in the dataIterable Props Array, so it gets added as example in to both the v-data-table and the v-data-iterable doc pages.

## Motivation and Context
This answers a frequent questions seen on the Discord channel. Such as:
* How do I change sort direction by default?
* How do I sort by a specific column?
* How do I set the Rows Per Page by default?
* How do I set/change the page the component is on?

Resolves https://github.com/vuetifyjs/vuetifyjs.com/issues/503

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:
- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [ ] My code follows the code style of this project.

### CODE REVIEW

**Comment by blalan05** [comment]
The 'integer' is not syntax highlighted, not sure if that can be improved. Also it seems like the Monorepo update changed the formatting of the example sections.

**Comment by blalan05** [comment]
Also I'm not sure why this pull contains 3 files, I only staged the map.js file.

File: packages/api-generator/src/map.js [hunk 1]
```diff
@@ -124,7 +124,13 @@ const sharedGridProps = [
 const dataIterableProps = [
   {
     name: 'pagination',
-    sync: true
+    sync: true,
+    example: {
+      descending: 'boolean',
+      sortBy: 'string',
+      rowsPerPage: '\'integer\' | \'-1 === All\'',
```
Comment (KaelWD) [inline packages/api-generator/src/map.js]:
```suggestion
      rowsPerPage: 'number // -1 for All',
```

File: packages/api-generator/src/map.js [hunk 2]
```diff
@@ -124,7 +124,13 @@ const sharedGridProps = [
...
+      page: 'interger'
```
Comment (KaelWD) [inline packages/api-generator/src/map.js]:
```suggestion
      page: 'number'
```

File: packages/api-generator/src/map.js [hunk 3]
```diff
@@ -124,7 +124,13 @@ const sharedGridProps = [
 const dataIterableProps = [
   {
     name: 'pagination',
-    sync: true
+    sync: true,
+    example: {
+      descending: 'boolean',
+      sortBy: 'string',
+      rowsPerPage: 'number // -1 for All',
```
Comment (jacekkarczmarczyk) [inline packages/api-generator/src/map.js]: Pretty sure Kael meant
```suggestion
      rowsPerPage: 'number', // -1 for All
```

Comment (KaelWD) [reply packages/api-generator/src/map.js]: nope

**Comment by blalan05** [comment]
@KaelWD does this pass review now?

**Comment by KaelWD** [comment]
There's also `totalItems`, not sure what it's for though.

**Comment by blalan05** [comment]
Setting Total when using server-side?

**Comment by KaelWD** [comment]
Added in e5b8e6fe70e37981f5f36a6a8e6d2b81681fd5f1, also fixed some css with 9c7e0f3ac26e94cc47eeb6c6131b2cf6260f8757

**Pull request merged**